gradoop-flink/src/main/java/org/gradoop/flink/model/impl/functions/epgm/filters/And.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.flink.model.impl.functions.epgm.filters;
+
+import org.apache.flink.api.common.functions.FilterFunction;
+
+/**
+ * A filter that combines multiple filters using a logical {@code and}.
+ * This filter will therefore accept all objects that are accepted by
+ * every component filter.
+ *
+ * @param <T> The type of objects to filter.
+ */
+public class And<T> extends AbstractRichCombinedFilterFunction<T> {
```

Comment:
Also add direct unit tests for this class with calling constructor like `new And<>(alwaysTrue, alwaysTrue, alwaysTrue)` etc. .. same for `Or<>` and `Not<>`.